source/vecs/entitymanager.d

```diff
@@ -1367,8 +1388,11 @@ private:
 	assert(*position == Position(1, 4));
 	assert(!uintegral);
 
-	assert(*world.getOrEmplaceComponent!uint(entity, 45) == 45);
-	assert(*world.getOrEmplaceComponent!uint(entity, 64) == 45);
+	int* i; ulong* ul;
```

Comment:
> - One statement per line.

Please follow a more D style approach unless conveniently necessary. I do think this makes code less readable.

Reply:
This is only for unit testing the returned components, and it's above the `AliasSeq` defined below. I found it more pleasant because it follows the `AliasSeq` syntax and their use is just for one assert call.